README.md

```diff
@@ -1,2 +1,54 @@
-# idm-styleguide
-IDM's living styleguide
+# idm-web-uikit
+
+IDM's living styleguide.
```

Comment:
Change this line to: `IDM's living wen UI kit` to match `package.json` description field.